shell/platform/linux/fl_engine.cc

```diff
@@ -39,15 +39,35 @@ struct _FlEngine {
   // Thread the GLib main loop is running on.
   GThread* thread;
 
+  // The project this engine is running.
   FlDartProject* project;
+
+  // Renders the Flutter app.
   FlRenderer* renderer;
+
+  // Messenger used to communicate with engine.
```

Comment:
```suggestion
  // Messenger used to communicate with the engine.
```

BTW, should this be to communicate with the Dart app?

Reply:
I've updated it to match the docs on FlBinaryMessenger.